iceoryx_utils/README.md

```diff
@@ -1,11 +1,11 @@
 
-# iceoryx utils
+# Eclipse iceoryx utils overview
```

Comment:
If you want to be completely grammatically correct, then overview should be capitalized.

Reply:
Thanks for clarifying this. I'm always wondering, in newspaper articles, titles are usually capitalised. Had a look on some project documentations and there only the first word was capitalised, hence I chose the same. Happy to adapt it, if you think it's a major thing